test/e2e/layers/mapMLTemplatedFeatures.test.js

```diff
@@ -61,4 +61,21 @@ describe("Playwright mapMLTemplatedFeatures Layer Tests", () => {
       expect(feature).toEqual("M307 456L599 467L612 629L381 599z");
     });
   });
-});
\ No newline at end of file
+  describe("Simple query by select values without map extent filter tests", () => {
+    test("All features loaded at start", async () => {
+      const features = await page.$$("css= body > map:nth-child(2) > .mapml-web-map >> css= div > .mapml-templatedlayer-container > div > div > svg > g > g");
+      expect(features.length).toEqual(8);
+    });
+    test("User can select/filter features by category", async () => {
```

Comment:
@ahmadayubi I am having a time getting this test to run.  The first challenge is to expose the select list via a series of layer control actions, then to evaluate the number of features that result from one or two selections.